src/SumoCoders/FrameworkCoreBundle/Resources/doc/development/pagination.md

```diff
@@ -0,0 +1,70 @@
+# Using pagination
+
+Pagination is a nice way to handle large amounts of data over multiple pages.
+
+## Controller
+
+In the controller the following code should be used. If necessary a custom adapter can be used.
+
+```
+<?php
+use Symfony\Bundle\FrameworkBundle\Controller\Controller,
+    Symfony\Component\HttpFoundation\Request,
+    Sensio\Bundle\FrameworkExtraBundle\Configuration\Method,
+    Sensio\Bundle\FrameworkExtraBundle\Configuration\Route,
+    Sensio\Bundle\FrameworkExtraBundle\Configuration\Template,
+    Pagerfanta\Pagerfanta,
+    Pagerfanta\Adapter\DoctrineORMAdapter,
+    Pagerfanta\Exception\NotValidCurrentPageException;
+class ExampleController extends Controller
```

Comment:
I'm missing an empty line between the use statements and the class

Reply:
It's also not according to our style.

Reply:
fixed 82caa0e